libs/hdf-converters/src/base-converter.ts

```diff
@@ -137,12 +142,15 @@ function collapseDuplicates<T extends object>(
 }
 
 export class BaseConverter {
-  data: Record<string, unknown>;
+  unconvertedData: Record<string, unknown>;
```

Comment:
Might want to call this source data, seems more professional

Reply:
i mean nominally this is all source data cause we're not generating data ourselves (except in those cases where we are where we supply default nist tags, etc).  i find 'unconverted' to be a more accurate descriptor.  we can be less concerned about things seeming "professional" since end users are not gonna see that term in any hdf files.